tests/eko/test_g_functions.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# Test G functions implemented by Muselli PhD
+
+import numpy as np
+
+from eko.anomalous_dimensions import harmonics as h
+
+zeta3 = h.constants.zeta3
+log2 = h.constants.log2
+
+
+testN = [1, 10, 100]
+
+# compare the exact values of some harmonics with Muselli parametrization
+def test_g3():
+    ns = [1.0, 2.0, 1 + 1j]
+    # NIntegrate[x^({1, 2, 1 + I} - 1) PolyLog[2, x]/(1 + x), {x, 0, 1}]
+    mma_ref_values = [0.3888958462, 0.2560382207, 0.3049381491 - 0.1589060625j]
```

Comment:
Stupid consistency remark: what here is called `mma_ref_values` everywhere else is `refvals`